### ISSUE TITLE
TWiLight Menu++ 3DS

### ISSUE THREAD

**Issue by Proyecto-N** [Universal-Team/pkmn-chest#19]
Will the program in ".nds" (or ".cia", explain if it works) work in the TWiLight Menu++ of the 3DS?

**Comment by SuperSaiyajinVoltZ** [comment]
You can download the `.nds` File and put it to the root of your SD Card and use it with TWiLightMenu++. If you want to be able to update `pkmn-chest`, you can also install `Universal-Manager` (another Project from our Team). Short answer : `Yes, the .nds File will work with TWiLightMenu for 3DS.`

**Comment by SuperSaiyajinVoltZ** [comment]
also : The `.cia` works as well, but you Need the `.nds` File as well into the root or in `SD:/_nds/pkmn-chest/`.

**Comment by Proyecto-N** [comment]
And I can you transfer game-exclusive Pokémon? Ex: Zekrom to Pokemon Black (Pokémon White exclusive)

**Comment by SuperSaiyajinVoltZ** [comment]
Yes you can, because Zekrom is also in the Pokédex of Pokémon Black. You can Transfer everything, which is available in the Game.
